fdbclient/NativeAPI.actor.cpp

```diff
@@ -6935,6 +6936,10 @@ Future<Void> Transaction::commitMutations() {
 		}
 		return commitResult;
 	} catch (Error& e) {
+		if (e.code() == error_code_transaction_throttled_hot_shard) {
+			TraceEvent(SevInfo, "TransactionThrottledHotShard").detail("Error", e.what());
```

Comment:
Nit: 
```suggestion
			TraceEvent("TransactionThrottledHotShard").suppressFor(1.0);
```
This event is unique, so we don't need the error code.